packages/pn-personagiuridica-webapp/src/services/configuration.service.ts

```diff
@@ -80,7 +81,6 @@ export function getConfiguration(): PgConfiguration {
     IS_DEVELOP,
     MOCK_USER: IS_DEVELOP,
     LOG_REDUX_ACTIONS: IS_DEVELOP,
-    URL_FE_LOGOUT: `${configurationFromFile.URL_FE_LOGIN}logout`,
```

Comment:
restore this line but replate it with `${configurationFromFile.SELFCARE_BASE_URL}/auth/logout`

After this rework, URL_FE_LOGIN will be useless. Remove all its occurences